### ISSUE TITLE
Memory size?

### ISSUE THREAD

**Issue by timriker** [whscullin/apple2js#215]
It looks like there is a 384k memory card in slot 2 when in //e emulation.

Can the aux slot configuration be configured to max memory? Emulating something like this:

https://www.tindie.com/products/garrettswrkshp/ram2e-ii-gw4203b-8mb-ram-for-apple-iie/

**Comment by leanto** [comment]
Currently there is no UI for setting the size of the RAM card, but you can change it in code.

If you look at [`js/main2e.ts` line 69](https://github.com/whscullin/apple2js/blob/main/js/main2e.ts#L69), you can see that it actually sets up a 1M RAMFactor-compatible card. You should be able to to just change that `16`  and have a 16 MB card. (I have not tried this.)

If you're using the new fancy preact UI (from `index.html`), then you need to change [`Slinky.tsx` line 24](https://github.com/whscullin/apple2js/blob/main/js/components/Slinky.tsx#L24).

**Comment by iflan** [comment]
Currently there is no UI for setting the size of the RAM card, but you can change it in code.

If you look at [`js/main2e.ts` line 69](https://github.com/whscullin/apple2js/blob/main/js/main2e.ts#L69), you can see that it actually sets up a 1M RAMFactor-compatible card. You should be able to to just change that `16`  and have a 16 MB card. (I have not tried this.)

If you're using the new fancy preact UI (from `index.html`), then you need to change [`Slinky.tsx` line 24](https://github.com/whscullin/apple2js/blob/main/js/components/Slinky.tsx#L24).

**Comment by timriker** [comment]
Thanks!

I'm not sure what the difference is, as I don't own either, but the board I linked to goes in the AUX slot instead of slot 2 as //jse is configured. Under AppleWin with -r 127 there is no RAMFactor visible, but there is still 8M of ram available.

I seem to recall there is software that won't work well with 16M and only supports 8M, or 127 "pages" in the ram expansion.

Is there a way to add 8M to the AUX slot?

It might be nice to have the default //e image online configured that way.

**Comment by whscullin** [comment]
Yeah, we can probably bump that. The existing limit was based on old browser behaviors and the amount of storage required  to serials the contents for saving and restoring state (which currently only partly works but is a ongoing goal).
